sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -56,9 +56,6 @@ public async Task CreateAndGetAzureApplicationInsightsDataFeedWithMinimumSetup(b
             DataFeed createdDataFeed = disposableDataFeed.DataFeed;
 
             ValidateDataFeedWithMinimumSetup(createdDataFeed, dataFeedName);
-
-            Assert.That(createdDataFeed.SourceType, Is.EqualTo(DataFeedSourceType.AzureApplicationInsights));
-
             ValidateAzureApplicationInsightsDataSource(createdDataFeed.DataSource as AzureApplicationInsightsDataFeedSource);
```

Comment:
All these `SourceType` assertions have been moved to `Validate<...>DataSource` methods, since we can get it directly from the `DataFeedSource` instance.